<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TWinTuner</class>
 <widget class="QDialog" name="TWinTuner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры приложения</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IconsForMenu.qrc">
    <normaloff>:/1/Picture/Tools.png</normaloff>:/1/Picture/Tools.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="CheckProto">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открывать протокол автоматически</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="CheckMap">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Открывать карту автоматически</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="IconsForMenu.qrc">
         <normaloff>:/1/Picture/OKGreen.png</normaloff>:/1/Picture/OKGreen.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>38</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
       <property name="icon">
        <iconset resource="IconsForMenu.qrc">
         <normaloff>:/1/Picture/OKRed.png</normaloff>:/1/Picture/OKRed.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="IconsForMenu.qrc"/>
 </resources>
 <connections/>
</ui>
